### ISSUE TITLE
In a Cataclysm build that uses "home_dir", and /usr/ as a prefix, where do the mod tilesets go?

### ISSUE THREAD

**Issue by Bracket-H** [Theawesomeboophis/UndeadPeopleTileset#40]
Alright, I'm done experimenting with this on my own, so I'm asking for a spoonfeeding here.

Here are my structures
/home/myself/.cataclysm-dda/
/home/myself/.cataclysm-dda/config
/home/myself/.cataclysm-dda/font
/home/myself/.cataclysm-dda/graveyard
/home/myself/.cataclysm-dda/memorial
/home/myself/.cataclysm-dda/mods/(currently containing the UndeadPeople tileset mods like DinoMod, but they're not showing up)
/home/myself/.cataclysm-dda/save
/home/myself/.cataclysm-dda/sound
/home/myself/.cataclysm-dda/templates

/home/myself/.cataclysm-dda/UndeadPeopleTileset/(git repo)

/home/myself/.cataclysm-dda/gfx/
/home/myself/.cataclysm-dda/gfx/UndeadPeople (symlinked to ~/.cataclysm-dda/UndeadPeopleTileset/TILESETS/gfx/MSX++UnDeadPeopleEdition/) It works and regular game tiles are graphical, and use this tileset.

/usr/share/cataclysm-dda/core/
/usr/share/cataclysm-dda/credits/
/usr/share/cataclysm-dda/font/
/usr/share/cataclysm-dda/gfx/
/usr/share/cataclysm-dda/help/
/usr/share/cataclysm-dda/json/
/usr/share/cataclysm-dda/mods/ (the game mods are in here, not the graphical mods)
/usr/share/cataclysm-dda/motd/
/usr/share/cataclysm-dda/names/
/usr/share/cataclysm-dda/raw/
/usr/share/cataclysm-dda/sound/
/usr/share/cataclysm-dda/title/

There is no data directory for me, so I don't know where to put the graphical mods so they show up in the worldgen, and ultimately would show the graphical tiles in the game.
DinoMod, for example, shows letters, not graphical tiles (although technically the letters are graphical tiles, but placeholders)

Alledgedly, graphical mods should go in 
./data/mods
?
but I don't have a data direction, CDDA reads directly from
/usr/share/cataclysm-dda and ~/.cataclysm-dda

Edit:
It works if I merge the contents of the graphical mod into the game mod, but that is kind of a crap way to maintain things...
If that is the way to go though, then you can close this issue.

**Comment by Theawesomeboophis** [comment]
Can you put this in Layman's terms please, I don't really understand the question.

**Comment by Bracket-H** [comment]
I don't know how to put this in other words, ^^; maybe someone else can help? 
But feel free to close this issue at your leisure.

**Comment by Theawesomeboophis** [comment]
I'll try getting someone to help.

**Comment by ProjectileWaterfowl** [comment]
Try putting it in /usr/share/cataclysm-dda/mods/. Usually all mods go in data/mods, so if the mods that would normally be located there are in cataclysm-dda/mods/ then the graphical ones should fit there too.

**Comment by Theawesomeboophis** [comment]
Does the proposed solution work, need to know if this is ready for closing or if issues are still occurring.

**Comment by Theawesomeboophis** [comment]
Gonna close this, If you're still getting issues, ping me and I'll reopen until fixed.
